packages/server/modules/core/repositories/users.ts

```diff
@@ -508,9 +524,32 @@ export const lookupUsersFactory =
 
     // limit to given project
     if (projectId) {
+      // Workspace implicit roles logic:
```

Comment:
fix for users() query not supporting implicit workspace roles

Reply:
when looking up project collaborators in mentions & i think invite dialogs as well?